Tiny tweaks to Reolink docs (#35938)
home-assistant/home-assistant.github.io: source/_integrations/reolink.markdown

3. Select the chime you want to remove from the list of devices
4. Underneath the **Device info**, select the three-dot {% icon "mdi:dots-vertical" %} menu. Then, select **Delete**.

> This will also decouple the chime from the doorbell in the reolink app/client. Therefore the chime will not ring anymore when the doorbell is pressed.

Reviewer: ```suggestion
This will also decouple the chime from the doorbell in the Reolink app/client. Therefore, the chime will no longer ring when the doorbell is pressed.
```
